clang/test/CodeGenOpenCL/sampled_image.cl

```diff
@@ -0,0 +1,12 @@
+// RUN: %clang_cc1 %s -triple x86_64-unknown-linux-gnu -O0 -emit-llvm -o - | FileCheck %s
```

Comment:
Is it intention to test it with `x86` triple?

Reply:
yes-ish, it follows the same patterns as for `image.cl`. This is just to make the output stable when it comes to running the tests.